AndroidSDKPush/src/main/java/com/leanplum/LeanplumLocalPushListenerService.java

```diff
@@ -34,24 +37,30 @@
  *
  * @author Aleksandar Gyorev
  */
-public class LeanplumLocalPushListenerService extends IntentService {
-  public LeanplumLocalPushListenerService() {
-    super("LeanplumLocalPushListenerService");
-  }
+public class LeanplumLocalPushListenerService extends JobIntentService {
+
+    public static Intent getIntent(Context context) {
```

Comment:
Can you please help me understand why we are implementing getIntent and passing the static content as extra data?

Reply:
It is just a helper method to construct the Intent needed to start this service.